adhocracy4/contrib/views.py

```diff
@@ -18,3 +19,20 @@ def raise_exception(self):
         limited by the current phase.
         """
         return self.request.user.is_authenticated()
+
+
+class SortableListView(generic.ListView):
+    """
+    ordering_current: List of possible ordering tuples (ordering,
+        ordering name).
+    """
+    orderings_supported = []
+
+    def get_ordering(self):
+        ordering = self.request.GET.get('ordering')
+        if ordering and ordering in dict(self.orderings_supported):
+            self.ordering = [ordering]
+        return self.ordering
```

Comment:
Getter functions should not have side effects. I think we should either set `self.ordering` in the beginning of the request (`__init__`?) or not set it at all and always use `get_ordering()`.